export_malloc/to_export.c

```diff
@@ -0,0 +1,15 @@
+#include <stdio.h>
+#include <emscripten/emscripten.h>
+
+void EMSCRIPTEN_KEEPALIVE fillData(int32_t *buffer,size_t size){
```

Comment:
will an exported function like this work? Won't the pointer be converted to i32 in wasm?

Reply:
What does this function mean?